ddtrace/logger.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+
+def configure_ddtrace_logger():
```

Comment:
can you add a docstring and type hinting to this function?

```suggestion
def configure_ddtrace_logger():
    # type: () -> None
    """
    """
```